scripts/ci/7-check-docs.sh

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env bash
+
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+set -e
+
+# Go to the Airflow project root directory
+FWDIR="$(cd "`dirname "$0"`"/../..; pwd)"
+cd "$FWDIR"
+
+# Install airflow
+pip install -e .[doc]
+
+# Go to docs folder
+cd docs
+
+NUM_IGNORE_WARNINGS=4
+NUM_CURRENT_WARNINGS=$(bash build.sh |\
+    tee /dev/tty |\
+    grep "build succeeded" |\
+    sed  -r 's/build succeeded, ([[:digit:]]+) warnings\./\1/g')
+
+if [ "${NUM_CURRENT_WARNINGS}" != "${NUM_IGNORE_WARNONGS}" ]; then
+    echo
+    echo "Unexpected problems found in the documentation. "
+    echo "Currently, ${NUM_IGNORE_WARNINGS} warnings are ignored."
+    echo "Please correct your PR. Thank you for your contribution to the project."
+    echo
+    exit 1
```

Comment:
Maybe we should just put this login in docs/builds.sh? Thoughts?

Reply:
Sounds good. This will be better for the developer who creates the documentation, because he will know if the change meets the project standards or not immediately after compilation.